lib/exec.ex

```diff
@@ -204,4 +184,50 @@ defmodule QueryParser.Exec do
       end
     )
   end
+
+  defp exec_value_operator?("$eq", value, elem_value, elem, ctx) do
+    elem_value == exec_value(value, elem, ctx)
+  end
```

Comment:
```suggestion
  defp exec_value_operator?("$eq",  elem_value, value) do
    elem_value == value  end
```
Same on other exec_value_operator?